### ISSUE TITLE
Add Scroll to Azuro 

### ISSUE THREAD

**Issue by V498** [scroll-tech/contribute-to-scroll#773]
### Name of the project

Azuro protocol

### Project type

Tool

### Site of the project

https://azuro.org

### Github repository of the project

https://github.com/Azuro-protocol

### Networks

Scroll Mainnet

### Additional context

Azuro is an infrastructure/liquidity layer for on-chain betting. It utilizes a new Liquidity Pool design, the Liquidity Tree, which creates and maintains market liquidity.

Being able to Providing Liquidity in Azuro protocol on the Scroll network would be a very huge one and would be so welcomed by many.

Azuro supports thousands of sports markets & other games, plus a rich feature set, which can be utilized by anyone permissionlessly for the creation of various apps, embedded integrations or derivative products.

One of people's favorite Betting Dapp is the Bookmaker.xyz which is the First Decentralized
Betting Site In The World.

In conclusion, It'd really be so Awesome having scroll integrated into Azuro Protocol.
